open-api/rest-catalog-open-api.yaml

```diff
@@ -212,6 +212,34 @@ paths:
           schema:
             type: string
           example: "accounting%1Ftax"
+        - name: pageToken
+          in: query
+          description:
+            Allows clients to make use of pagination by signaling to the service that they would
+            prefer requests to be paginated to a “reasonable” number of results.
```

Comment:
> a “reasonable” number of results.

or a number specified by pageSize? and why double quote reasonable?

Reply:
maybe "a number of results that the REST service considers reasonable"?

Reply:
I think we also need `required: false` and `allowEmptyValue: true` here.

The description should also indicate that an empty value should signal to paginate and follow up requests should include the continuation token provided by the server.

Reply:
Ack will reword this.